scripts/run-mongodb.ts

```diff
@@ -1,49 +1,23 @@
-import { MongoMemoryServer } from 'mongodb-memory-server';
```

Comment:
So we don't test when it's not a replica set ? Do we stop supporting it ?

Reply:
IMO we don't, because Maggregor is based on Mongo events and a standalone MongoDB doesn't support the changeStreams (https://www.mongodb.com/docs/v5.2/changeStreams/#availability).
We have to properly answer to this question in https://github.com/maggregor/maggregor/issues/40